src/qml/controls/ToggleButton.qml

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+Button {
+    property color defaultColor: Theme.color.neutral2
+    property color hoverColor: Theme.color.neutral2
+    property color activeColor: Theme.color.neutral5
+    property color textColor: Theme.color.neutral7
+    property color textHoverColor: Theme.color.orangeLight1
+    property color textActiveColor: Theme.color.neutral9
+    id: root
+    checkable: true
+    hoverEnabled: true
+    font.family: "Inter"
+    font.styleName: "Regular"
+    font.pixelSize: 13
+    leftPadding: 12
+    rightPadding: 12
+    topPadding: 5
+    bottomPadding: 5
+
+    contentItem: Text {
```

Comment:
We could just make this a Header and we can remove specifying font in lines 18-20, and this becomes:

```qml
    contentItem: Header {
        id: buttonText
        header: root.text
        headerSize: 13
        headerColor: root.textColor
    }
```

Reply:
Does it make sense to just have a "CoreText" item?